app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -44,4 +44,20 @@ def textual_product
   def textual_version
     @record.properties['Version']
   end
+
+  def textual_lives_on
+    lives_on_ems = @record.try(:lives_on).try(:ext_management_system)
+    return nil if lives_on_ems.nil?
+    lives_on_entity_name = _("Virtual Machine")
+     {
+         :label      => "Underlying #{lives_on_entity_name}",
```

Comment:
Actually I read it the opposite way - underlying vm = the virtual machine on which the middleware server is running.
This is also how it is done in containers.
I have no objection to get couple of alternatives from reviewers ("runs on" as you suggested is an option), and then see if it can be aligned in containers too. @serenamarie125 @Catrobson @simon3z